JS/dom.js

```diff
@@ -0,0 +1,170 @@
+
+const header = document.querySelector('header .header-nav');
+let plusOneBtn = document.querySelector('.PlusOne');
+let minusOneBtn = document.querySelector('.MinusOne');
```

Comment:
A good practice to follow is to declare all your variables with const key word unless you intend to re-assign it somewhere else in the code then use the let key word.